iam_assumerole/data.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_iam_policy" "permission_boundary_policy" {
+  name = "PermissionBoundaryPolicy"
```

Comment:
This should be a variable, e.g. `var.permission_boundary_policy_name`, in cases where the parent module doesn't have a boundary policy ***explicitly*** named `PermissionBoundaryPolicy`.

If it needs to be that specific name, please add a note in the `README`, as well as a comment here, indicating that this external policy must exist before using/creating this module (otherwise, Terraform will fail attempting to read from a policy that doesn't exist).

Reply:
Made the changes as requested. Please review.